lib/astarte_data_access/config.ex

```diff
@@ -27,6 +27,19 @@ defmodule Astarte.DataAccess.Config do
 
   use Skogsra
 
+  @type ssl_option ::
+          {:cacertfile, String.t()}
+          | {:verify, :verify_peer}
+          | {:server_name_indication, :disable | charlist()}
+          | {:depth, integer()}
+  @type ssl_options :: :none | [ssl_option]
+  @type auth_options :: {module(), [{String.t(), String.t()}]}
+  @type xandra_opts :: Xandra.start_options()
```

Comment:
Use `Xandra.start_options()` directly in the specs instead of defining another type